get_prism_data.R

```diff
@@ -4,39 +4,232 @@ library(prism)
 library(raster)
 library(dplyr)
 library(tidyr)
+library(stringr)
 options(prism.path = "./data/prismdata")
 
-months <- c(1:12)
-clim_vars <- c("ppt", "tmin", "tmean", "tmax")
-for (month in months){
-  for (clim_var in clim_vars){
-    get_prism_monthlys(type=clim_var, year = 1981:2014, month = month, keepZip=F)
+#Years of prism data to 
+yearsToUse=1981:2014
+#Load the DB. create = TRUE does not seem 
+
+sqliteDBFile='./data/bbsforecasting.sqlite'
+if(file.exists(sqliteDBFile)){
+  database <- src_sqlite(sqliteDBFile, create = FALSE)
+} else {
+  database <- src_sqlite(sqliteDBFile, create = TRUE)
+}
```

Comment:
I think we can just use `database <- src_sqlite(sqliteDBFile, create = TRUE)` with none of the conditional logic. The documentation is unclear on this point, but a quick test suggests that `create` only comes into play if the file doesn't exist already (and this would be a pretty common approach to this sort of API).